.github/workflows/pull_request.yaml

```diff
@@ -20,6 +20,10 @@ on:
         default: false
 
 jobs:
+  helm-lint:
```

Comment:
Why does this run even if there isn't any `helm` folder?

Reply:
@shimoncohen it's smart to detect whether there's a directory from the top of the tree which includes a helm chart , if there isn't it won't continue and it'll skip

Reply:
From what I see this runs even if there isn't a `helm` folder. I think we should add the ability to disable this `job` or at least detect here if the folder exists before running.
We should also add a parameter indicating the root folder for the charts defaulting to `helm`.

Reply:
@shimoncohen changed it to run conditionally

Reply:
Awesome, what about passing a custom helm root folder? This should happen in the workflow itself and not in this file.

Reply:
What about `helm-charts` repo for example? And again, don't assume this will be true for all repos, I think we should support custom root folder.

Reply:
@shimoncohen added an input and moved it from the config